src/components/ResultsTable.tsx

```diff
@@ -278,6 +326,25 @@ const ResultView = ({
               ) : (
                 cell(k)
               )}
+              {i < columns.length - 1 && (
```

Comment:
Non intuively, this means the dragging surface area is slightly left of the border. I plan to iterate on that along with any other feedback tomorrow